packages/discord_gcp_alerts/lib/src/models/gcp_alert.dart

```diff
@@ -0,0 +1,244 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'gcp_alert.g.dart';
+
+/// {@template gcp_alert}
+/// A GCP alert notification.
+/// https://console.cloud.google.com/monitoring/alerting/notifications
+/// {@endtemplate}
+@JsonSerializable()
+class GCPAlert {
```

Comment:
Are there docs for the object this is deserializing? Linking to those in the comments would be helpful.

Reply:
I wish there were...I derived this by reverse engineering the notifications but would love to be pointed to proper documentation.